tools/proto-js-plugin/src/test/resources/build.gradle

```diff
@@ -76,5 +76,6 @@ protobuf {
 build.dependsOn compileProtoToJs
 
 dependencies {
+    protobuf "io.spine:spine-base:${spineVersion}"
```

Comment:
This seems wrong, as it references the "previously" published artifact, not the current one.

You can refer to `base` protos via filesystem if you need those to be taken into account during their generation. If you are not completely sure, let's discuss.